chaosazure/vmss/actions.py

```diff
@@ -242,8 +247,21 @@ def choose_vmss_instance_at_random(vmss_choice, configuration, secrets):
         logger.debug(
             "Found virtual machine scale set instances: {}".format(
                 [x['name'] for x in vmss_instances]))
-    logger.warning("Temporarily hard coding to the first instance which sould be zone 0")            
-    choice_vmss_instance = vmss_instances[0]
+    logger.warning("Temporarily hard coding to the first instance which sould be zone 0")
+    choice_vmss_instance = random.choice(vmss_instance)
+    return choice_vmss_instance
+
+def choose_vmss_instance(vmss_choice, configuration, secrets, vmss_name):
```

Comment:
missing typing annotations :)

Reply:
I wasn't sure on this... there were a bunch of other methods around it that didn't have it either.

Reply:
I need to find ways to automate that constraint so I can appreciate why you were confused. Would you mind adding them whenever you add new functions?